homework9/src/components/footer/Footer.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+const Footer = () => {
+  return (
+    <footer>
+      <div className="footer--container">
+        <div>
+          <p>©Simple Art, Inc. 2021</p>
+          <p>All rights reserved</p>
+        </div>
+        <ul className="footer-nav">
+          <li>
```

Comment:
here u have repeated code for every `li`
create some separate component for that

Reply:
Thanks